// Example showcasing try-catch-finally with the specialized keywords

prep {
    ingredient _x := 0;
    serve("Started");
    eightysix("Error Entered", TypeError);
} rescue (TypeError e) {
    serve(e.stack);
} cleanup {
    serve("Finished");
}

'''
rescue (ValueError v) {
    serve(v.stack);
} 
'''
